.gitignore

```diff
@@ -6,3 +6,5 @@ node_modules
 *.DS_Store
 logs
 *.log
+deviceconfig.js
+*.vscode
```

Comment:
Should this be deviceconfig.json ? JSON files can still be loaded directly as node modules.